site/soc/software/charts/osh/openstack-compute-kit/nova.yaml

```diff
@@ -83,9 +83,29 @@ data:
         monitors: {{ ','.join(suse_airship_deploy_ceph_mons) }}
         admin_keyring: {{ ceph_admin_keyring_b64key | b64decode }}
         cinder:
-          user: {{ ses_cluster_configuration.cinder.rbd_store_user }}
+          user: {{ ses_cluster_configuration['cinder']['rbd_store_user'] }}
```

Comment:
was the old style broken or new style is preferred?

Reply:
I saw that osh team is using that style and thought it should be consistent with that code.  But I see many places to be fixed in our code. so will keep our style.